libbeat/_meta/fields.ecs.yml

```diff
@@ -1569,6 +1569,12 @@
           description: >
             Process parent id.
 
+        - name: pgid
```

Comment:
I would suggest for now that we added this to the metricbeat fields.yml instead of the ECS one so we only have it in metricbeat.

Reply:
Agreed